frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -7,7 +7,8 @@ pods:
     placement: "{{PLACEMENT_CONSTRAINT}}"
     {{#ENABLE_VIRTUAL_NETWORK}}
     networks:
-      dcos:
+      {{VIRTUAL_NETWORK_NAME}}:
```

Comment:
No? Because we want virtual networks to be false by default, but we want the default virtual network to be `dcos`